source/devices/AM64X/linux/Release_Specific_Release_Notes.rst

```diff
@@ -36,36 +36,34 @@ found on the SDK download page or in the installed directory as indicated below.
 -  Linux Manifest:  :file:`<PSDK_PATH>/docs/software_manifest.html`
 
 
-Release 10.00.07.04
+Release 10.01.10.04
 ===================
 
-Released on August 2024
+Released on December 2024
 
 What's new
 ----------
 
-**Processor SDK Linux AM64X Release has following Major Updates**
+**Processor SDK Linux AM64X Release has following new features:**
+
+  - LTS Stable Kernel update to 6.6.58
+  - Important Bug Fixes on top of Processor SDK 10.00.07.04 Release
+  - RT Kernel : Real-Time Linux Interrupt Latency numbers here - :ref:`RT Interrupt Latencies <RT-linux-performance>`
 
-  - LTS Refresh with Stable Kernel update to 6.6.32
-  - Important Bug Fixes on top of Processor SDK 09.02.01.10 Release
-  - Kernel: MMC Highspeed tuning and Performance Improvements
-  - RT Kernel : Significant Improvements to Real-Time Linux Interrupt Latency - :ref:`RT Interrupt Latencies <RT-linux-performance>`
-  - Bug Fixes and Stability Improvements
-  - TIFS, DM, R5 SPL: PLL Configuration sequence updated to latest recommendation
-  - TISCI ABI version updated to 4.0 from 3.1 (Low power mode related API changes - LPM does not apply to AM64x)
-  - HSR Tx Tag Insertion and Rx Tag Removal offload on ICSSG
 
 **Component version:**
 
-  - Kernel 6.6.32
-  - RT Kernel 6.6.32-rt32
+  - Kernel 6.6.58
+  - RT Kernel 6.6.58-rt45
   - U-Boot 2024.04
   - Toolchain GCC 13.3
-  - ATF 2.10+
-  - OPTEE 4.2.0
-  - TIFS Firmware v10.00.08
-  - DM Firmware 10.00.00.08
-  - Yocto scarthgap
+  - ATF 2.11+
+  - OPTEE 4.4.0
+  - Graphics DDK 24.1
```

Comment:
Is this applicable for AM624x which doesn’t has GPU?

Reply:
good catch , copy paste error .